bin/fuel-core/src/cli/run.rs

```diff
@@ -268,15 +262,14 @@ impl Command {
 
         // if consensus key is not configured, fallback to dev consensus key
         let consensus_key = load_consensus_key(consensus_key)?.or_else(|| {
```

Comment:
Could you also move the code below here?

```rust
        if consensus_key.is_some() && trigger == Trigger::Never {
            warn!("Consensus key configured but block production is disabled!")
        }
```

We need to print it only in the case if the `load_consensus_key` returns something